File: tests/fixtures/code_samples/java/sample_016.java
// Sample 16: small utility.
package samples;

import java.util.List;

public final class Sample016 {
    private Sample016() {}

    public static int operation(List<Integer> xs) {
        int total = 16;
        for (int x : xs) total += x;
        return total;
    }

    public static int operationPure(int v) {
        return (v * 16) %% 7919;
    }
}

File: tests/fixtures/code_samples/ts/sample_016.ts
/** Sample 16: small utility. */

export function operation16(xs: number[]): number {
  let total = 16;
  for (const x of xs) total += x;
  return total;
}

export function operationPure16(value: number): number {
  return (value * 16) % 7919;
}

